packages/create-instance/create-component-stubs.js

```diff
@@ -86,31 +86,48 @@ export function createStubFromComponent(
   // ignoreElements does not exist in Vue 2.0.x
   if (Vue.config.ignoredElements) {
     Vue.config.ignoredElements.push(tagName)
+    if (Vue.config.ignoredElements.indexOf('template-stub') === -1) {
+      Vue.config.ignoredElements.push('template-stub')
+    }
   }
 
   return {
     ...getCoreProperties(componentOptions),
     $_vueTestUtils_original: originalComponent,
     $_doNotStubChildren: true,
     render(h, context) {
-      return h(
-        tagName,
-        {
-          attrs: componentOptions.functional
-            ? {
-                ...context.props,
-                ...context.data.attrs,
-                class: createClassString(
-                  context.data.staticClass,
-                  context.data.class
-                )
-              }
-            : {
-                ...this.$props
-              }
-        },
-        context ? context.children : this.$options._renderChildren
+      const attrs = componentOptions.functional
+        ? {
+            ...context.props,
+            ...context.data.attrs,
+            class: createClassString(
+              context.data.staticClass,
+              context.data.class
+            )
+          }
+        : {
+            ...this.$props
+          }
+
+      const slots = context ? context.slots() : this._renderProxy.$slots
+
+      // ensure consistent ordering of slots (default first, then alphabetical)
+      const sortedSlotEntries = Object.entries(slots)
+      sortedSlotEntries.sort(([slotNameA], [slotNameB]) =>
+        slotNameA === 'default'
+          ? -1
+          : slotNameB === 'default'
+          ? 1
+          : slotNameA.localeCompare(slotNameB)
+      )
+
+      const children = sortedSlotEntries.map(([slotName, slotChildren]) =>
```

Comment:
So this should render the default slot, and all others as children of the `template-stub`? Wont this be a super breaking change for snapshots?

Reply:
True, this will break all snapshots.

Probably not the biggest deal, we just need to make sure that's clear in release notes. Something like:

NOTE: You will need to update all snapshots when updating.